docs/available_opds.rst

```diff
@@ -1,22 +1,13 @@
-.. JWST-PSFs documentation master file, created by
-   sphinx-quickstart on Mon Nov 29 15:57:01 2010.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
-
-
 Appendix: Available Optical Path Difference (OPD) files
 ================================================================
 
-
 For each of the five instruments (four SIs plus FGS) there are three provided OPD files. These represent wavefronts as follows:
 
 1. The OTE and ISIM intrinsic WFE
 2. The above, plus a slight defocus to blur the image slightly to approximate image motion. 
 3. The above #2, plus additional WFE due to SI internal optics. 
 
-The latter is the largest WFE, and is the default file used in simulations unless another is explicitly chosen. For NIRCam only there is a second, duplicate 
-set of these files with slightly improved WFE based on an optimistic case scenario for instrument and telescope alignment. 
+The latter is the largest WFE, and is the default file used in simulations unless another is explicitly chosen. For NIRCam only there is a second, duplicate set of these files with slightly improved WFE based on an optimistic case scenario for instrument and telescope alignment.
```

Comment:
Great, added! (Just noticed that the table wasn't compiling, so I fixed that too. Oops.)